### ISSUE TITLE
does this work for elasticsearch 1.7.3?

### ISSUE THREAD

**Issue by trekr5** [spinscale/elasticsearch-graphite-plugin#17]
Hi,

Hoping someone can help.

I've installed your plugin on a node running elasticsearch 1.7.3 with the following snippet from my elasticsearch.yml file

    plugin.mandatory: graphite
    metrics.graphite.host: graphite.dev.aws.mycompany.service
    metrics.graphite.port: 2003
    metrics.graphite.prefix: elasticsearch

and I can't see any metrics in graphite.

Two questions: -

1) Does my configuration look correct?
2) Does this plugin work for elasticsearch 1.7.3?

Many thanks

**Comment by spinscale** [comment]
yes it should, please check your logfiles for error logs and startup information, anything in there
